ExpenseTracker/ExpenseTracker.Application/Mappings/WalletMappings.cs

```diff
@@ -33,4 +33,13 @@ public static WalletViewModel ToViewModel(this Wallet wallet)
         Owner = null!,
         OwnerId = request.UserId,
     };
+    public static Wallet ToUpdateEntity(this UpdateWalletRequest request) => new()
+    {
+        Id= request.Id,
```

Comment:
Id =

add space between Id and equal sign "="

Reply:
problem solved